src/Common/DNSResolver.cpp

```diff
@@ -179,7 +176,7 @@ struct DNSResolver::Impl
     using HostWithConsecutiveFailures = std::unordered_map<String, UInt32>;
     using AddressWithConsecutiveFailures = std::unordered_map<Poco::Net::IPAddress, UInt32>;
 
-    CacheBase<std::string, DNSResolver::IPAddresses> cache_host{100};
+    CacheBase<std::string, DNSResolver::CacheEntry> cache_host{100};
```

Comment:
By the way as I see `CacheBase` accepts number of bytes, correct me if I', wrong.
If yes, since value size became bigger , will it make amount of records records less? Also `100` bytes seems really small, how much records do we actually have in practice?

Reply:
I added the setting for this